### ISSUE TITLE
'gnt-cluster upgrade' command should stop the watcher

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#961]
Originally reported of Google Code with ID 905.

```
What software version are you running? Please provide the output of "gnt-
cluster --version", "gnt-cluster version", and "hspace --version".
2.10.1

<b>What distribution are you using?</b>
Ubuntu Precise

<b>What steps will reproduce the problem?</b>
1. Run gnt-cluster upgrade --to 2.10
2. Interrupt the command after the 'stopping daemons everywhere' step
3. Observe daemons being restarted on nodes by the ganeti watcher

What is expected?
As per the upgrade document [1], the manual instructions included pausing the watcher as the second step.  The new automated command doesn't seem to do this, which results in daemons being restarted on nodes if the upgrade procedure takes longer than the watcher interval (5mins?).

I think the "automatized" upgrade should also stop the watcher.

[1] http://docs.ganeti.org/ganeti/2.11/html/upgrade.html
```
Originally added on 2014-07-31 15:24:33 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-08-01 07:13:52 +0000 UTC.

Added Labels: Type-Defect Priority-High
Changed State: Accepted
Added to Milestone: Release2.10

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-08-01 07:14:03 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
(reassigning, as I won't get to it any time soon. There's a patch out that needs more work, feel free to use and adapt this. Thx!)
```
Originally added on 2014-08-12 07:24:39 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 49fd9730e44fa20dd04c815f861dfeefd840dc7e
Author: Klaus Aehlig <aehlig@google.com>
Date:   Wed Aug 13 10:59:43 2014 +0200

    Pause watcher during upgrade
    
    As also suggested in the manual upgrade instructions,
    pause the watcher (for 1h) during the upgrade. Fixes
    issue 905.
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Helga Velroyen <helgav@google.com>

```
Originally added on 2014-08-13 10:52:24 +0000 UTC.

Mentioned Issue with GoogleCode ID 905 has been migrated to GitHub with ID #961.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.11.6
```
Originally added on 2014-09-22 15:56:40 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.11.6
```
Originally added on 2014-09-22 15:57:10 +0000 UTC.

Changed State: Released
